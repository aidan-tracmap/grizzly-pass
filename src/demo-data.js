// @flow

import type { Project, Label } from "./types";

export const projects: Project[] = [
  /*
  {
    id: "1",
    title: "Coffee Swirl",
    person: "Joe Lemon",
    time: "2017-03-15T10:54:04.445Z",
    progress: 13,
    status: "ontrack",
    labels: ["TMOL"]
  },
  {
    id: "2",
    title: "Rake Twister",
    person: "Alex Apple",
    time: "2017-04-12T10:54:04.445Z",
    progress: 50,
    tentative: true,
    status: "onhold",
    labels: ["TMOL", "Em"]
  }
  */
];

export const labels: Label[] = [
  {
    id: "TMOL",
    title: "TMOL"
  },
  {
    id: "Em",
    title: "Embedded"
  }
];
