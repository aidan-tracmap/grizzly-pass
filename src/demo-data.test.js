import { projects } from "./demo-data";

test("project demo data", () => {
  expect(projects).toMatchSnapshot();
});
