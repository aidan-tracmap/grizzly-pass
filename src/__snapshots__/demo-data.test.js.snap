exports[`test project demo data 1`] = `
Array [
  Object {
    "id": "1",
    "person": "Joe Lemon",
    "progress": 13,
    "status": "ontrack",
    "time": "2017-03-15T10:54:04.445Z",
    "title": "Coffee Swirl",
  },
  Object {
    "id": "2",
    "person": "Alex Apple",
    "progress": 50,
    "status": "onhold",
    "time": "2017-04-12T10:54:04.445Z",
    "title": "Rake Twister",
  },
]
`;
